### PR TITLE
✨ adapters: create adapter for nunjucks templating language

### PR DESCRIPTION
**Pull request by CosAnca**

### CODE REVIEW

**Comment by dennisreimann** [comment]
Can you remove the yarn cache files?

**Comment by CosAnca** [comment]
Done. Not sure how they "slipped" in there.

File: test/project/uiengine.config.js [hunk 1]
```diff
@@ -26,7 +27,7 @@ module.exports = {
     components: ['./src/elements', './src/modules', './src/examples'],
     templates: './src/templates',
     pages: './uiengine/pages',
-    additionalWatches: './lib'
+    additionalWatches: './lib',
```
Comment (dennisreimann) [inline test/project/uiengine.config.js:30]: Can you remove those trailing comma changes too?

**Comment by dennisreimann** [comment]
Looks good, will merge once the build works! Thank you :)

File: test/project/uiengine.config.js [hunk 2]
```diff
@@ -51,6 +52,24 @@ module.exports = {
       module: '@uiengine/adapter-webpack',
       options: webpackAdapterReactOptions
     },
+    njk: {
+      module: '/Users/cos/Playground/uiengine/packages/adapter-nunjucks',
```
Comment (dennisreimann) [inline test/project/uiengine.config.js:56]: I think this is why the tests fail :)

**Review by dennisreimann — CHANGES_REQUESTED** [review]

File: test/project/uiengine.config.js (hunk 2 above)
Comment (CosAnca) [reply test/project/uiengine.config.js]: I've updated the path to a relative path but the tests are still failing. Is there a specific pattern I need to follow to reference the module in here?

Comment (dennisreimann) [reply test/project/uiengine.config.js]: I think I fixed it with ebfb45f. There was also [this one](https://github.com/dennisreimann/uiengine/pull/85/commits/ebfb45ff3e195fc640fc1cf6f6e0871e9b1f95d9#diff-4b364b41b339d2f6ecd774d156ab1feea93189b6fdf33465f21a639854100e0e).

**Comment by dennisreimann** [comment]
Thank you very much, Cos! Published as v3.2.0 🎉

**Pull request merged**